x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawl_detail_logic.ts

```diff
@@ -81,16 +81,16 @@ export const CrawlDetailLogic = kea<MakeLogicType<CrawlDetailValues, CrawlDetail
     ],
   },
   listeners: ({ actions }) => ({
-    fetchCrawlRequest: async ({ requestId }) => {
+    fetchCrawlEvent: async ({ requestId }) => {
```

Comment:
The old name seems more correct because a process crawl is also considered an event.